dask_expr/_core.py

```diff
@@ -5,6 +5,7 @@
 import weakref
 from collections import defaultdict
 from collections.abc import Generator
+from typing import Literal, TypeAlias
```

Comment:
this doesn't work on py3.9. I think you need to use typing_extensions

Reply:
Fixed